packages/wallets/src/features/accounts/screens/DocumentSubmission/DocumentSubmission.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { Dropzone, useFlow, WalletText } from '../../../../components';
+import useDevice from '../../../../hooks/useDevice';
+import Upload from '../../../../public/images/accounts/upload.svg';
+import { getExampleImagesConfig } from '../../constants';
+import { CommonMistakesExamples } from '../CommonMistakesExamples';
+import './DocumentSubmission.scss';
+
+const ListItems = [
```

Comment:
```suggestion
const listItems = [
```
naming: capitalize only for component names